Special abilities v2, mostly Flak (#4)
GameBrains/er-core: modules/special-rules/pages/a-dying-race.adoc

= A Dying Race

> Every Eldar casualty is a tragedy, one step closer to the race's extinction.

Reviewer: ```suggestion
Every Eldar casualty is a tragedy, one step closer to the extinction of that race.
```

Author: Why 'that' instead of 'the'? It reads really awkwardly.

Reviewer: Mmmm...not sure, brain fart perhaps, or some half-remembered grammar technicality that may or may not be valid...
I'd be happy with "extinction of the race" — though I've had another idea, I'll amend my suggestion.

Reviewer: Note that it looks to me as if, when you resolve the conversation, the suggestion won't be available to apply - the buttons get greyed out.

---

> Apply these rules to detachments with the {a-dying-race} ability:

Reviewer: ```suggestion
Where your detachment has the {a-dying-race} ability, apply these rules:
```
I'm suggesting this formulation of the sentence to better support other changes below.  Although, if we only have one short bullet then it will probably be better to lose the bullet and combine this:
"Where your detachment has the {a-dying-race} ability and it is destroyed, remove D3 points from your army morale."

Author: Doh, it should have been the singular 'this rule' version. 🙄

I'm not happy about changing the whole sentence, because I'm trying to keep the ability construction as consistent as possible. Based on some of the existing abilities like Assault, Engineers etc, I've been working with this sentence construction, followed by one or more bullet points:

> Apply these rules/this rule to [subjects] with the {keyword-attribute} ability:

If you would like me to use a _different_ construction that's fine, but whatever construction is chosen should be used consistently. That's also why I've used the same construction even when there's only one effect - it's adding a level of visual consistency that helps comprehension and retention.

If you don't have an alternative standard construction,
```suggestion
Apply this rule to detachments with the {a-dying-race} ability:
```

Author: It's a Detachment ability, so yes it's going to be 'has the {x} ability'.

However, this is one of the Detachment abilities that could theoretically be written as an Army rule instead. I just thought it was better categorised as a Detachment ability due to it being resolved per-detachment.

Author: Suggestion added to commit batch.

> * If the detachment is wiped out, immediately remove D3 points from the player's army morale. [either/or]

Reviewer: ```suggestion
* If the detachment is destroyed, remove D3 points from your army morale.
```
"Wiped out" is a sort of colloquialism, so I think plain English 'destroyed' is better. Arguably, "If every unit in the detachment is destroyed..." might be better still, but I'm not sure.
I don't think we need the adverb "immediately" when we use this imperative form. Even more so if we were to replace "When..." with "If..." — though that does sort of imply that your Eldar **will** cop it... >-D

Author: 'Destroyed' is a specific type of result on the war engine Catastrophic Damage Charts, and is the result that gives the enemy half the WE's morale value as boost for seeing it blow up ([Compendium link, bullet 2](https://thehobby.zone/resources/e40k-compendium/Content/Battles/ScenarioSpecialRules/ArmyMorale.htm#link2)). So I don't think it's a safe word to use here. GW used 'wiped out' relatively frequently.

I did have a 'when the last unit in the detachment is killed' option, but it looked messy.

Some form of time reference is required, because the player's army morale drops as soon as they lose the detachment, while almost all other changes to army morale happen in the Rally phase.

Reviewer: Well, "GW uses it a lot" isn't exactly a high bar for writing standards, as we know. 'Wiped out' is an idiom (not colloquialism). Idioms (and colloquialisms) are okay and perhaps interesting for creative writing, but very bad practice in rules text.

"Destroyed" is in use for much more than just the Catastrophic Damage charts. Apart from general usage, in the rules usage there is frequent mention of units being destroyed. I think we need to be explicit about when we mean the specific damage chart result (and when 'Wrecked' does or doesn't count*) — not avoid the term for any other usage. So I strongly disagree that 'destroyed' is a problematic term. Even more so in my alternate construction "If every unit in the detachment is destroyed..."

_*Though we should be mindful of 'progressive disclosure' too, so as not to overwhelm newbies with advanced rules stuff._

Yeah, options like 'when the last unit in the detachment is killed' occurred to me too, but I do try to avoid that sort of thing — the more so as it is passive case. If we were forced to use a construction like that somewhere then "When you are forced to remove the last unit as a casualty" would be appropriate and consistent.

I see your point about the time reference given the exception from usual timing. As mentioned, 'when' instead of 'if' would work, but as I say, unfortunately that sorts of implies an inevitability...
Adverbs are frowned upon but hard to avoid sometimes, so rather than tie ourselves up in knots, let's keep 'immediately' in this case.

Reviewer: More properly, while we're at it:
```suggestion
* If the detachment is destroyed, immediately subtract D3 points from your army morale.
```

I can't agree with your idea that figurative text can be more clear than plain English. Idioms, colloquialisms, figurative terms -- these all demand that the reader make some sort of connection that plain English doesn't, and assume that the reader has English as their first language... and even that the reader has the same cultural background.

These things can be useful or interesting for persuasive copy (marketing, targeting a specific audience), or entertaining copy stories and such - but they're bad practice for rules and other instructional or reference content.

Author: We'll have to agree to disagree with this specific example, so use what you want and I'll grit my teeth about it looking 'ugly'.

Regarding languages, I work with wargames rules in Spanish and English, and sometimes French and German, and spend a lot of time discussing rules with people whose first language (and sometimes second language) isn't English. Raise/lower are commonly used and understood terms, especially in the context of morale.

Author: Suggestion added to commit batch.

> * If the detachment is wiped out, the player's army morale immediately goes down by D3 points. [either/or]

Reviewer: ```suggestion
```
Definitely not. Nasty, weak passive case and third person. Yuck. I feel dirty just reading it. >-D

Author: Based on the raise/lower idea:
```suggestion
* If the detachment is wiped out, immediately lower the player's army morale by D3 points.
```

Reviewer: I deleted this because we're working on the alternate construction in the preceding line, so we don't need it. My comment was just to highlight that this option would've been especially egregious on the passive case and third person front; not only "is destroyed" which we accept as expedient in many cases, but also "goes" and "the player's", which we can avoid through active/second person/imperative as in my suggestion for the alternate option in the preceding line.

> +[TODO: decide on the syntax for army morale changes. The original text used several different options, I suggest 'goes up by X points' and 'goes down by X points'.]

Reviewer: ```suggestion
```
I agree with your general idea, but never in passive case. The {active case, second person, imperative} approach is almost always better.

Author: OK, but you need to give me a version that works with {active case, second person, imperative}.

Maybe 'raise/lower'? For example:

> If the detachment is wiped out, immediately lower the player's army morale by D3 points.

Reviewer: I did: "If the detachment is destroyed, remove D3 points from your army morale."
Technically, "is destroyed" is still passive case, but this is one of the exceptions where passive case is actually better.
No need for raise/lower, add/remove (or perhaps add/deduct) is more plain, without the figurative implication.

Author: For what it's worth, the figurative implication is the whole point of suggesting raise/lower. There might not be a 'need' for them, but in some cases I find that plainer text is less appropriate as it can disguise the implications of what's happening by making the text too impersonal.

> +[TODO: decide on the syntax for army morale changes. The original text used several different options, I suggest 'goes up by X points' and 'goes down by X points'.]

Author: Adding a comment for army morale attention. This file and tyranid-objectives.adoc are the ones affected so far, they both have some variant bullet points.
